src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AuthorizationFunctionalTest.java

```diff
@@ -79,9 +79,9 @@
     public void setUp() {
         RestAssured.useRelaxedHTTPSValidation();
 
-        log.info("Configured S2S secret: " + s2sSecret.substring(0, 2) + "************" + s2sSecret.substring(14));
-        log.info("Configured S2S microservice: " + s2sName);
-        log.info("Configured S2S URL: " + s2sUrl);
+        //log.info("Configured S2S secret: " + s2sSecret.substring(0, 2) + "************" + s2sSecret.substring(14));
+        //log.info("Configured S2S microservice: " + s2sName);
+        //log.info("Configured S2S URL: " + s2sUrl);

```

Comment:
These were commented out instead of being removed as they may be useful for developers while working/debugging